hieroplatform/src/test/java/org/hiero/sketch/NextKSketchTest.java

```diff
@@ -172,4 +171,21 @@ public void testTopK4() {
         NextKList nkl = big.blockingSketch(nk);
         Assert.assertEquals(nkl.table.toString(), "Table, 1 columns, 10 rows");
     }
+
+    @Test
+    public  void TestTopK5() {
+        Table t = GetTable.testTable();
+        RecordOrder ro = new RecordOrder();
+        ro.append(new ColumnSortOrientation(t.getSchema().getDescription("Age"), true));
+        ro.append(new ColumnSortOrientation(t.getSchema().getDescription("Name"), true));
+        StringBuilder sb = new StringBuilder();
```

Comment:
Using a StringBuilder for this is overkill, just make it a long constant string.